pkgs/games/freedroid/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, stdenv
+, fetchFromGitHub
+, makeDesktopItem, copyDesktopItems
+, imagemagick
+, autoconf, automake
+, SDL, SDL_mixer, SDL_image, SDL_gfx
+, libvorbis
+, libjpeg, libpng
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "freedroid";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner  = "ReinhardPrix";
+    repo   = "FreedroidClassic";
+    rev    = "release-${version}";
+    sha256 = "027wns25nyyc8afyhyp5a8wn13x9nlzmnqzqyyma1055xjy5imis";
+  };
+
+  nativeBuildInputs = [ autoconf automake copyDesktopItems imagemagick ];
+  buildInputs = [
+    SDL SDL_image SDL_gfx
+    (SDL_mixer.override { enableMusicMod = true; })
+    libjpeg libpng libvorbis zlib
+  ];
+
+  postPatch = "touch NEWS";
+
+  preConfigure = "./autogen.sh";
+
+  postInstall = ''
+    mkdir -p $out/share/icons/hicolor/32x32/apps
+    convert graphics/paraicon.bmp $out/share/icons/hicolor/32x32/apps/freedroid.png
+  '';
+
+  desktopItems = [ (makeDesktopItem {
+    name = "${pname}";
+    exec = "${pname}";
+    icon = "${pname}";
+    desktopName = "Freedroid Classic";
+    comment     = "A clone of the classic game 'Paradroid' on Commodore 64";
+    categories  = "Game;ArcadeGame;";
+  }) ];
+
+  meta = with lib; {
+    description = "A clone of the classic game 'Paradroid' on Commodore 64";
+    homepage    = "https://github.com/ReinhardPrix/FreedroidClassic";
+    license     = licenses.gpl2;
+    maintainers = with maintainers; [ iblech ];
+    platforms   = platforms.unix;
+    broken      = stdenv.isDarwin;
```

Comment:
```suggestion
    # Builds but fails to render to the screen at runtime.
    broken      = stdenv.isDarwin;
```

Reply:
Noting that the failure is runtime rather than build time would help future maintainers.

Reply:
Oh, indeed, of course. Will be fixed in the next force-push!